neuralprophet/forecaster.py

```diff
@@ -1824,22 +1844,27 @@ def plot_latest_forecast(
             fcst, self.config_train.quantiles, n_last=1 + include_previous_forecasts
         )
 
-        # Check whether the default plotting backend is overwritten
+        # Check whether a local or global plotting backend is set.
         plotting_backend = (
-            plotting_backend
-            if plotting_backend != "default"
-            else (self.plotting_backend if hasattr(self, "plotting_backend") else "matplotlib")
+            auto_set_plotting_backend(plotting_backend)
```

Comment:
Suggestion: For ease of reading, you could use a more traditional if-else format.